web/src/app/shared/models/notifications.ts

```diff
@@ -56,254 +56,255 @@ export enum ERROR_TYPES {
+    '',
+    'Workbasket {this.workbasket.name} Distribution targets were saved successfully'
   )],
   // workbasket-information.component
-  [ERROR_TYPES.SUCCESS_ALERT_9, new Pair(
-    'DistributionTarget for workbasketID {this.workbasket.workbasketId} was removed successfully',
-    ''
+  [NOTIFICATION_TYPES.SUCCESS_ALERT_9, new Pair(
+    '',
+    'DistributionTargets for workbasketID {this.workbasket.workbasketId} was removed successfully'
   )],
   // workbasket-information.component
-  [ERROR_TYPES.SUCCESS_ALERT_10, new Pair(
-    'Workbasket {workbasketUpdated.key} was saved successfully',
-    ''
+  [NOTIFICATION_TYPES.SUCCESS_ALERT_10, new Pair(
+    '',
+    'Workbasket {workbasketUpdated.key} was saved successfully'
   )],
   // workbasket-information.component
-  [ERROR_TYPES.SUCCESS_ALERT_11, new Pair(
-    'Workbasket {workbasketUpdated.key} was created successfully',
-    ''
+  [NOTIFICATION_TYPES.SUCCESS_ALERT_11, new Pair(
+    '',
+    'Workbasket {workbasketUpdated.key} was created successfully'
   )],
   // workbasket-information.component
-  [ERROR_TYPES.SUCCESS_ALERT_12, new Pair(
-    'The Workbasket {workbasketId} has been deleted.',
-    ''
+  [NOTIFICATION_TYPES.SUCCESS_ALERT_12, new Pair(
+    '',
+    'The Workbasket {workbasketId} has been deleted.'
   )],
   // forms-validator.service
-  [ERROR_TYPES.WARNING_ALERT, new Pair(
-    'There are some empty fields which are required.',
-    ''
+  [NOTIFICATION_TYPES.WARNING_ALERT, new Pair(
+    '',
+    'There are some empty fields which are required.'
   )],
   // forms-validator.service x2
-  [ERROR_TYPES.WARNING_ALERT_2, new Pair(
-    'The {responseOwner.field} introduced is not valid.',
-    ''
+  [NOTIFICATION_TYPES.WARNING_ALERT_2, new Pair(
+    '',
+    'The {responseOwner.field} introduced is not valid.'
   )],
   // taskdetails.component
-  [ERROR_TYPES.DANGER_ALERT, new Pair(
-    'There was an error while updating.',
-    ''
+  [NOTIFICATION_TYPES.DANGER_ALERT, new Pair(
+    '',
+    'There was an error while updating.'
   )],
   // taskdetails.component
-  [ERROR_TYPES.SUCCESS_ALERT_13, new Pair(
-    'Task {this.currentId} was created successfully.',
-    ''
+  [NOTIFICATION_TYPES.SUCCESS_ALERT_13, new Pair(
+    '',
+    'Task {this.currentId} was created successfully.'
   )],
   // taskdetails.component
-  [ERROR_TYPES.SUCCESS_ALERT_14, new Pair(
-    'Updating was successful.',
-    ''
+  [NOTIFICATION_TYPES.SUCCESS_ALERT_14, new Pair(
+    '',
+    'Updating was successful.'
   )],
   // taskdetails.component
-  [ERROR_TYPES.DANGER_ALERT_2, new Pair(
-    'There was an error while creating a new task.',
-    ''
+  [NOTIFICATION_TYPES.DANGER_ALERT_2, new Pair(
+    '',
+    'There was an error while creating a new task.'
   )],
   // task-master.component
-  [ERROR_TYPES.INFO_ALERT_2, new Pair(
-    'The selected Workbasket is empty!',
-    ''
+  [NOTIFICATION_TYPES.INFO_ALERT_2, new Pair(
+    '',
+    'The selected Workbasket is empty!'
   )],
```

Comment:
Generally I don't like that we have the `this` reference within the template. This may lead to some confustion for less technical people. 

What do you think of renaming e.g. `this.workbasketId` to `workbasketId`. Same applies for anything nested.

Reply:
`responseOwner.field` can also be reduced to `owner`

Reply:
If you agree on doing this don't forget to replace all the replacement keys within the calling replacement maps

Reply:
I knew this, but wanted to refactor this together with the entire `NOTIFICATION_TYPES` at a later time. The `NOTIFICATION_TYPES` should get split in `MESSAGE_TYPES` and `ERROR_TYPES` and renamed to more appropriate and specific names.
e.g.: `NOTIFICATION_TYPES.SUCCESS_ALERT -> MESSAGE_TYPES.SUCCESS_ACCESS_REMOVE`
I have replaced the keys for now.